cns/configuration/env.go

```diff
@@ -11,13 +11,30 @@ const (
 	EnvNodeName = "NODENAME"
 	// EnvNodeIP is the IP of the node running this CNS binary
 	EnvNodeIP = "NODE_IP"
-	// LabelSwiftV2 is the Node label for Swift V2
-	LabelSwiftV2 = "kubernetes.azure.com/podnetwork-multi-tenancy"
+	// LabelNodeSwiftV2 is the Node label for Swift V2
+	LabelNodeSwiftV2 = "kubernetes.azure.com/podnetwork-multi-tenancy-enabled"
+	LabelPodSwiftV2  = "kubernetes.azure.com/pod-network"
+	EnvPodV4CIDR     = "POD_V4_CIDR"
```

Comment:
just out of curiosity, what's the reason for having these as env vars instead of in the cns configMap? these CIDRs should be shared between all nodes so it feels like they are fine to have in the configMap? cc @rbtr

Reply:
I probably said we could push them in to the env like downward API at some point and we just ran with that. 
I don't think there's a practical difference, in an ideal world (ie with viper) the env and the configmap would be equal and interchangeable :( do you have a preference? We could still put them in both places

Reply:
maybe weak preference towards configMap since I don't think we'd ever want different CNSes to have different CIDR sets and 2) if anything else in the future needed to read these values for whatever reason it's easier to read a cm than parse a ds spec for env vars.

really not going to die on this hill though so if we're too far into this to change that's fine with me.